lib/jetstream/api/kv.ex

```diff
@@ -151,6 +152,56 @@ defmodule Jetstream.API.KV do
     end
   end
 
+  @doc """
+  Get all the non-deleted values for a Key Value Bucket
+
+  ## Examples
+
+      iex>%{"key1" => "value1} = Jetstream.API.KV.all_values(:gnat, "my_bucket")
+  """
+  @spec all_values(conn :: Gnat.t(), bucket_name :: binary()) :: %{}
+  def all_values(conn, bucket_name) do
```

Comment:
The name `all_values` reminds me of `Map.values` from the standard library, but that function only returns a list of values without their associated keys. 

Does NATS documentation have a term for getting the non-deleted entries from a key-value bucket? Maybe this is already named after that documentation?

Reply:
What about `contents(conn, bucket)`? That way you're getting the non-deleted contents of a bucket?

Reply:
That's a good point. Thanks for pointing that out. I like `contents` better.

Reply:
@mmmries updated here: https://github.com/mmmries/jetstream/pull/64/commits/3f65e5be4920f97c5a9ee86c6547dd1cbc32dc13

Reply:
This looks like https://hexdocs.pm/elixir/Map.html#to_list/1. I have also thought about word `entries`.